docs/source/conf.py

```diff
@@ -62,18 +62,59 @@
 ]
 
 intersphinx_mapping = {
-    "python": ("https://docs.python.org/3.7", "python37.inv"),
+    "python": ("https://docs.python.org/3.7", None),
     "numpy": ("https://docs.scipy.org/doc/numpy/", None),
     "scipy": ("https://docs.scipy.org/doc/scipy/reference/", None),
-    "cirq": ("https://cirq.readthedocs.io/en/stable/", None),
-    "pyquil": ("http://docs.rigetti.com/en/stable/", None),
+    # Cirq no no longer using sphinx docs so interlinking is not possible.
+    #"cirq": ("https://quantumai.google/cirq", None),
+    "pyquil": ("https://pyquil-docs.rigetti.com/en/stable/", None),
     "qiskit": ("https://qiskit.org/documentation/", None),
     "qutip": ("http://qutip.org/docs/latest/", None),
 }
 
 
 # source_suffix = '.rst'
-source_suffix = [".rst", ".md"]
+source_suffix = {
+    ".rst": "restructuredtext",
+    ".ipynb": "myst-nb",
+    ".myst": "myst-nb",
+    ".mystnb": "myst-nb",
+}
+
+myst_enable_extensions = [
+    "amsmath",
+    "colon_fence",
+    "deflist",
+    "dollarmath",
+    "html_image",
+]
+myst_url_schemes = ("http", "https", "mailto")
+
+# https://myst-nb.readthedocs.io/en/latest/use/formatting_outputs.html#render-priority
+nb_render_priority = {
+    "html": (
+        "application/vnd.jupyter.widget-view+json",
+        "application/javascript",
+        "text/html",
```

Comment:
I guess we just need to have these without too many questions.

Reply:
I'll add some comments on what they are for, but basically are there to help configure MyST parser.